enclave-manager/api/typescript/package.json

```diff
@@ -1,11 +1,11 @@
 {
+    "version": "0.0.0",
     "name": "enclave-manager-sdk",
     "main": "./build/index",
     "description": "This repo contains a Typescript client for communicating with the Enclave Manager API server.",
     "types": "./build/index",
     "scripts": {
-        "build": "tsc && cp -R src/ build/ && cp -R node_modules/kurtosis-sdk/src/core/kurtosis_core_rpc_api_bindings/connect/* build/ && cp -R node_modules/kurtosis-sdk/src/engine/kurtosis_engine_rpc_api_bindings/connect/* build/",
-        "test": "ts-mocha -p tsconfig.json 'test/**/*.ts'"
```

Comment:
We don't have an tests but should we leave this to make it possible to run them?

Reply:
I've added back a stub in case anything wants to call `yarn test` in the future.

I don't think we should leave `"test": "ts-mocha -p tsconfig.json 'test/**/*.ts'"` by default as to a reader it looks like it does something, when in fact we know that currently it does nothing.

Reply:
I see, yes it could be misleading. I do prefer keeping it so that when we _do_ add unit tests they will automatically be picked up and run as part of the build script (also not giving extra burden to others to get unit tests going). This is pattern we have in other places too (especially our go projects): we look for test files and run them if they exist. This means all build scripts are basically identical in function. 

if you know of a better test command to run then please do feel free to change it.

Reply:
ACK

I've added Jest as the testing library and a basic test to the package - I'm not sure the tests will ever get more complicated that this for this package (instead integration tests of the enclave manager frontend will be more appropriate).

Reply:
Perfect. Love this!